pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "powersploit";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-cFj4IwDyiwQDSYLCcloMQ83T8MOtMcKsyMilfrZOyHs=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/powersploit
+    cp -a --no-preserve=ownership * $out/share/powersploit
```

```suggestion
    cp -a * $out/share/powersploit
```

no need to change, just noting that nix changes ownership of files anyways

---

Okay, thanks for information, I'll keep that in mind for my next derivations.